js-operators-coercion/operatorsAndCoercion.js

```diff
@@ -0,0 +1,109 @@
+// Створіть змінну яка буде символізувати вік людини. 
+
+const personAge = 18;
+
+/* Після чого створіть булеву змінну, яка буде визначати чи особа досягла віку коли може отримати права,
+( в Україні це 18 років), базуючись на заченні з попередньої змінної. Результат виведіть у консоль. */
+
+let isPersonAllowed = personAge >= 18;
+console.log(isPersonAllowed);
+
+/* 2.Створіть дві змінні, які містять імена двох осіб. 
+Використовуючи конкатенацію рядків, створіть третю змінну, яка містить вітання для обох осіб. Виведіть результат в консоль.
+Базуючись на результаті всіх змінних зробіть конкантенацію всіх рядків (4 змінна, або просто обєднати у консоль лозі все) і виведіть результат у консоль. */
+
+let nameOne = 'Olena';
+let nameTwo = 'Serhiy';
```

Comment:
Your file is written using double quotes, pls change the quotes. 
The best practice is to choose either single or double quotes.